frontend/fingertips-frontend/components/organisms/SpineChartTable/SpineChartTableStyles.tsx

```diff
@@ -76,37 +75,39 @@ export const StyledAlignRightBorderRightTableCell = styled(
   paddingRight: '0.5em',
 });
 
-export const StyledAlignCentreBorderRightTableCell = styled(
-  StyledAlignCentreTableCell
+export const StyledAlignRightCellPadLeft = styled(StyledAlignRightTableCell)({
+  paddingLeft: '0.5em',
+});
+
+export const StyledValueUnitStickyCell = styled(
+  StyledAlignRightBorderRightTableCell
 )({
-  borderRight: 'solid #bfc1c3 1px',
-  paddingRight: '0.5em',
+  ...(stickyLeft as unknown as TemplateStringsArray),
+  left: `${spineChartIndicatorTitleColumnMinWidth + spineChartPeriodColumnMinWidth + paddingSize * 2}px`,
+});
+
+export const StyledPeriodStickyCell = styled(StyledAlignCentreTableCell)({
+  ...(stickyLeft as unknown as TemplateStringsArray),
+  minWidth: `${spineChartPeriodColumnMinWidth}px}`,
```

Comment:
```suggestion
  width: `${spineChartPeriodColumnMinWidth}px}`,
```
table does some weird auto resizing for me if minWidth is used here